README.md

```diff
@@ -5,16 +5,31 @@ We hope the process is going well for you so far.
 
 Please follow the instructions below, and don't hesitate to ask us for anything you need.
 
+The dev task is currently available in the following programming languages:
+* Python (both `the_big_bang.py` and `the_big_bang.ipynb` are available)
+* Go (`big_bang.go` and `big_bang_test.go`)
+* Node.js (`big_bang.js`)
+
+
 ## Instructions
 
+If you prefer to develop locally: 
+1. Clone the repo
+2. Choose the relevant file based on your preferred programming language
+3. Follow the instructions in the file
+
+
+If **Python** is your favorite language, and you prefer working with Jupyter Notebook, you can also use **Binder**:
 1. Go to [https://mybinder.org/](https://mybinder.org/)
 
 2. Set [https://github.com/SygniaLabs/backend-dev-task](https://github.com/SygniaLabs/backend-dev-task) as the GitHub repository URL and hit launch
 
-3. Open `the_big_bang.ipynb` and follow the instructions there. There are also stub versions for Node (big_bang.js) and Go (big_bang.go), you can choose based on your preferred programming language.
+3. Open `the_big_bang.ipynb` and follow the instructions there
 
-4. If for some reason you are having hard time working with Binder, you can always download `the_big_bang.py` and work locally in your favorite IDE!
 
-5. Once you are done, download the notebook file as .ipynb, or take the relevant go/js file and send it to: `michael.h@sygnia.co`
+Once you are done:
+1. Download the notebook file as .ipynb, or take the relevant py/go/js file
+2. Zip the file
+3. Mail it to: `michael.h@sygnia.co`
```

Comment:
Can you ask Sapir to create an email group for this instead?
That way we can control it easily and add/remove people as we see fit (like our tower support group for example)

Reply:
Great idea! Will do.